corehq/apps/commtrack/models.py

```diff
@@ -193,6 +157,7 @@ class CommtrackConfig(QuickCachedDocumentMixin, Document):
     multiaction_enabled = BooleanProperty()
     multiaction_keyword_ = StringProperty()
 
+    # todo: remove?
     requisition_config = SchemaProperty(CommtrackRequisitionConfig)
     openlmis_config = SchemaProperty(OpenLMISConfig)
```

Comment:
yeah i think we can remove all of it. i thought it was already removed, but sounds like i didn't give danny complete instructions: https://trello.com/c/dcfnPhR0/14-openlmis-integration-code